packages/common/db/createDbConfig.ts

```diff
@@ -1,25 +1,21 @@
-export type DbConfig = {
-  host: string
-  port: number
-  username: string
-  password: string
-  max: number
-  idle_timeout: number
-  max_lifetime: number
-  ssl: boolean | { rejectUnauthorized: false }
-  onnotice: () => boolean
-}
+import { baseConfig } from "./baseConfig"
 
-const createDbConfig = (): DbConfig => ({
-  host: process.env.DB_HOST ?? "localhost",
-  port: Number(process.env.DB_PORT ?? "5432"),
-  username: process.env.DB_USER ?? "bichard",
-  password: process.env.DB_PASSWORD ?? "password",
-  ssl: process.env.DB_SSL === "true" ? { rejectUnauthorized: false } : false,
-  max: 10,
-  idle_timeout: 20,
-  max_lifetime: 60 * 30,
-  onnotice: () => false
-})
+// Note: we are declaring both user and username properties
+// in the config object.This is because the pg-promise
+// library expects the username property to be present
+// in the config object. However, the baseConfig object
+// uses the user property. To avoid any confusion,
+// we are declaring both properties in the createDbConfig
+// function.
+
+const createDbConfig = () =>
+  ({
+    ...baseConfig,
+    username: baseConfig.user,
+    max: 10,
+    idle_timeout: 20,
+    max_lifetime: 60 * 30,
+    onnotice: () => false
+  }) as const
```

Comment:
Why do we need the `as const`?

Reply:
This was used as suggested by Copilot to ensure type safety. The object would conform exactly as expected